utils/rtty/files/rtty.config

```diff
@@ -0,0 +1,6 @@
+config device
+    option interface 'lan'
+
+config server
+  	option	host 'jianhuizhao.f3322.net'
+  	option	port '5912'
```

Comment:
Maybe the `interface` option can also reside in the `server` section?

Reply:
Sample config should not refer to any specific server and should be off by default.